tools/PI/DevHome.PI/Models/ClipboardMonitor.cs

```diff
@@ -126,29 +112,24 @@ private ClipboardContents ParseClipboardContents(string text)
             }
 
             newContents.Raw = original;
-            newContents.Hex = errorAsInt is not null ? Convert.ToString((int)errorAsInt, 16) : original;
-            newContents.Dec = errorAsInt is not null ? Convert.ToString((int)errorAsInt, 10) : original;
+            newContents.Hex = Convert.ToString((int)errorAsInt, 16);
+            newContents.Dec = Convert.ToString((int)errorAsInt, 10);
 
-            // Is there an error code on here?
-            // if (ErrorLookupHelper.ContainsErrorCode(text, out var hresult))
-            if (errorAsInt is not null)
+            var errors = ErrorLookupHelper.LookupError((int)errorAsInt);
+            if (errors is not null)
             {
-                var errors = ErrorLookupHelper.LookupError((int)errorAsInt);
-                if (errors is not null)
+                foreach (var error in errors)
                 {
-                    foreach (var error in errors)
+                    // Seperate each error with a space. These errors aren't localized, so we may not need to worry
+                    // about the space being in the wrong place.
+                    if (newContents.Code != string.Empty)

```

Comment:
NIT: Stringbuilder instead of remaking strings.

Reply:
Good suggestion.

In this case, it's pretty uncommon to have the same error number correspond to multiple error strings, so my fear would be using a stringbuilder would actually be more expensive in this case. I'll keep an eye on it though.